src/client.py

```diff
@@ -0,0 +1,303 @@
+        assert all(isinstance(name, str) for name in [sceneName, wid]), \
+            "Parameters 'sceneName' and 'wid' must be strings"
+        
+        if self.__getSceneIndex(sceneName) == -1:
+            logger.error(f"addSceneToWindow(): Unknown sceneName '{sceneName}'.")
+            return False
+        elif wid not in self.windowList:
+            logger.error(f"addSceneToWindow(): Unknown windowName '{wid}'.")
+            return False
+        
+        index = self.__getSceneIndex(sceneName)
+        if index != -1:
+            rec = rr.new_recording(application_id=wid, recording_id=sceneName, spawn=True)
+            self.sceneList[index].setRec(rec)
+            return True
+        logger.error(f"Failed to find sceneName '{sceneName}' in scene list.")
+        return False
+    
+    def __parse_entity(self, entityName, entity, entityType):
+        """
+        Parse entity name and log (or not) entity :
+            - if there is a scene specified in entityName :  <scene>/name
+                it will log directly the entity into the scene
+            - if there is a '/' : <not a scene>/name
+                it will need addToGroup() to be log in a scene
+                every '/' will interpreted as a tree
+            - if there is no '/', entity will require addToGroup() to be logged
+        """
+        assert entity is not None, "__parse_entity(): 'entity' must not be None"
+        assert isinstance(entityType, Archetype), \
+            "__parse_entity(): 'entityType' must be of type `enum Archetype`"
+        
+        charIndex = entityName.find('/')
+        # if '/' in entityName then search for the scene in self.sceneList
+        if charIndex != -1 and charIndex != len(entityName) - 1:
+            sceneIndex = self.__getSceneIndex(entityName[:charIndex])
+            if sceneIndex != -1:
+                entity = Entity(entityName[charIndex + 1:], entity)
+                self.entityList[entityType.value].append(entity)
+                if entityType != Archetype.MESH_FROM_PATH:
+                    rr.log(entityName[charIndex:], entity.archetype, recording=self.sceneList[sceneIndex].rec)
+                else:
+                    rr.log_file_from_path(file_path=entity.archetype.path)
+                msg = (
+                    f"__parse_entity() creates a {entityType.name} for '{entityName}', "
+                    f"and logs it directly to '{self.sceneList[sceneIndex].name}' scene."
+                )
+                logger.info(msg)
+                return
+        # put entity to entityList, wait for addToGroup() to be logged
+        entity = Entity(entityName, entity)
+        self.entityList[entityType.value].append(entity)
+        msg = (
+            f"__parse_entity() does not create a {entityType.name} for '{entityName}', "
+                "it will be created when added to a group with addToGroup()."
+        )
+        logger.info(msg)
+    
+    def __get_entity(self, entityName):
```

Comment:
```suggestion
    def _get_entity(self, entityName):
```

One underscore: hidden things
Two underscores: python reserved stuff